### DISCUSSION TITLE
BedInteraction

### DISCUSSION THREAD

**Discussion by ArifJanMC** [Zrips/CMI#2281] [Feature Requests]
Is it possible to allow players set bedhome when it's not night with printing a message in chat?

**Comment by sandwally** [comment]
In config search for BedInteraction and change it to true.
And in Locale_EN look for sethome and you can set a message.
I think that will fix it.

**Comment by ArifJanMC** [comment]
Are you joking? I've already tested it. It's not working when it's day. You should read my message one more time to see it.

**Comment by Remski01** [comment]
You asked if thats a option, he explained the option.

**Comment by ArifJanMC** [comment]
Ok, but he didn't check it for the latest plugin version where it's not working.

**Comment by mercurialmusic** [comment]
It definitely works during the day for me...
